homework/latte_machiato/coffee_latte.py

```diff
@@ -0,0 +1,27 @@
+from homework.new_coffee.coffee import Coffee
+
+
+class CoffeeLatte(Coffee):
+
+    """
+    Coffee class with a bit of milk
+    """
+
+    def __init__ (self, latte_spoons, latte_sugar, latte_water, milk):
+        super().__init__(coffee_spoons=latte_spoons, sugar=latte_sugar, water=latte_water)
+        self.milk = milk
+
+    def prepare_coffee(self):
+        return Coffee.quantity + self.milk
+
+    def prepare_time_coffee(self):
+        time_to_prepare = self.water/10 + (self.coffee_spoons*10 + self.sugar*5)/5 + self.milk/10
```

Comment:
pep8 recommends to leave spaces between operators.
https://www.python.org/dev/peps/pep-0008/#other-recommendations